### DISCUSSION TITLE
Expand Submenus upon click/tap

### DISCUSSION THREAD

**Discussion by patzobil** [docmost/docmost#173] [Ideas]
It would be useful to have the option in the settings to enable automatic expansion of the left hand navigation when it a user clicks on the category to reveal the sub categories under that item

**Comment by Neoheresy** [comment]
> It would be useful to have the option in the settings to enable automatic expansion of the left hand navigation when it a user clicks on the category to reveal the sub categories under that item

On top of that it would be great to be able to choose to have the left menu fully expanded on login, as the default view.

**Comment by triplenick** [comment]
I think my suggestion is related to this one. 

So, I guess it's very crucial for a proper workflow to have a **history whichever pages were previously expanded or not**, at least on browser's localstorage level. It might be optional in user's preferences in case if it overloads the app.
